src/language-css/printer-postcss.js

```diff
@@ -853,6 +856,20 @@ function genericPrint(path, options, print) {
           continue;
         }
 
+        if (isMapItemNode) {
+          const isPartOfValue =
+            node.groups?.[1]?.type === "value-colon" && i > 1;
+          if (
```

Comment:
Can you add an example here to show what this condition supposed to match?

Reply:
Actually, I mean example for the code in this condition, not sure those `value-string`/`value-word` suppose to match.

Reply:
What do you think the following comment?

```js
// For example, there is the following key-value pair:
//
//   "xs-only": "only screen and (max-width: #{map-get($grid-breakpoints, "sm")-1})"
//
// "only screen and (max-width: #{map-get($grid-breakpoints, " is a "value-string"
// and "sm" is a "value-word"
// We should not insert any spaces and lines here.
```